modules/travel_pay/app/services/travel_pay/claims_service.rb

```diff
@@ -59,11 +60,23 @@ def get_claim_details(claim_id)
 
       @auth_manager.authorize => { veis_token:, btsss_token: }
       claim_response = client.get_claim_by_id(veis_token, btsss_token, claim_id)
+      documents = []
+      if include_documents?
+        begin
+          documents_response = documents_client.get_document_ids(veis_token, btsss_token, claim_id)
+          documents = documents_response.body['data'] if documents_response.body['data']
+        rescue
+          # Because we're appending documents to the claim details we need to rescue and return the details,
+          # even if we don't get documents
+          documents = []
```

Comment:
Agreed this should not blow up the overall behavior, but can we log the error w/in the rescue?